UnityProject/Assets/Scripts/Items/Pipes/PipeItem.cs

```diff
@@ -15,13 +15,13 @@ public class PipeItem : NetworkBehaviour, ICheckedInteractable<HandApply>, IChec
 
 		public SpriteHandler SpriteHandler;
 		public RegisterItem registerItem;
-		private PlayerRotatable rotatable;
+		public Rotatable rotatable;
 
 		private void Awake()
 		{
 			SpriteHandler = this.GetComponentInChildren<SpriteHandler>();
 			registerItem = this.GetComponent<RegisterItem>();
-			rotatable = GetComponent<PlayerRotatable>();
```

Comment:
Did you remove the component?

Reply:
nah, it doesn't make a difference which one this interacts with, I was trying this since apparently was null when being constructed, but fixed that